staking/availability/measure.go

```diff
@@ -193,68 +166,63 @@ func ComputeCurrentSigning(
 	signed, toSign :=
 		new(big.Int).Sub(statsNow.NumBlocksSigned, snapSigned),
 		new(big.Int).Sub(statsNow.NumBlocksToSign, snapToSign)
-	leftToGo := blocksPerEpoch - toSign.Uint64()
 
 	computed := staking.NewComputed(
-		signed, toSign, leftToGo, numeric.ZeroDec(), true,
+		signed, toSign, 0, numeric.ZeroDec(), true,
```

Comment:
hmm, why take it out, people wanted it, will affect RPC

Reply:
i moved it outside the func. it's only for RPCs. for protocol calculation, it's not needed.

Reply:
to avoid wiring unnecessary things into the protocol code.